pkgs/tools/audio/openai-whisper-cpp/default.nix

```diff
@@ -0,0 +1,60 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, SDL2
+, makeWrapper
+, wget
+, Accelerate
+, CoreGraphics
+, CoreVideo
+}:
+
+stdenv.mkDerivation rec {
+  pname = "whisper.cpp";
```

Comment:
Ah, there's one small thing, here — you name the binary `whisper-cpp`, not `whisper.cpp`, meaning `nix run` doesn't just work. I suspect the pname should be `whisper-cpp` (and then you'll have to change the `repo` to be `whisper.cpp`.